tests/funcalign/test_srm.py

```diff
@@ -117,7 +117,53 @@ def test_can_instantiate():
         s.fit(X)
     print("Test: different number of samples per subject")
 
+def test_new_subject():
+    import brainiak.funcalign.srm
+    s = brainiak.funcalign.srm.SRM()
+    assert s, "Invalid SRM instance!"
+
+    import numpy as np
+    np.random.seed(0)
+
+    voxels = 100
+    samples = 500
+    subjects = 2
+    features = 3
+
+    s = brainiak.funcalign.srm.SRM(n_iter=5, features=features)
+    assert s, "Invalid SRM instance!"
+
+    # Create a Shared response S with K = 3
+    theta = np.linspace(-4 * np.pi, 4 * np.pi, samples)
+    z = np.linspace(-2, 2, samples)
+    r = z**2 + 1
+    x = r * np.sin(theta)
+    y = r * np.cos(theta)
+
+    S = np.vstack((x, y, z))
 
+    X = []
+    W = []
+    Q, R = np.linalg.qr(np.random.random((voxels, features)))
+    W.append(Q)
+    X.append(Q.dot(S) + 0.1*np.random.random((voxels, samples)))
+
+    for subject in range(1, subjects):
+        Q, R = np.linalg.qr(np.random.random((voxels, features)))
+        W.append(Q)
+        X.append(Q.dot(S) + 0.1*np.random.random((voxels, samples)))
+
+    # Check that runs with 2 subject
+    s.fit(X)
```

Comment:
You should test that it does not succeed when the model is not computed before you call fit()